WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java

```diff
@@ -250,7 +250,10 @@ public int getItemViewType(int position) {
                 postView = LayoutInflater.from(context).inflate(R.layout.reader_cardview_xpost, parent, false);
                 return new ReaderXPostViewHolder(postView);
             case VIEW_TYPE_REMOVED_POST:
-                postView = LayoutInflater.from(context).inflate(R.layout.reader_cardview_removed_post, parent, false);
+                LayoutInflater inflater = LayoutInflater.from(context);
+                postView = mReaderImprovementsFeatureConfig.isEnabled()
+                        ? inflater.inflate(R.layout.reader_cardview_removed_post_new, parent, false)
```

Comment:
Nitpick: would it make sense to get the `LayoutRes` with the ternary operator and have a single line calling `inflater.inflate(<res>, parent, false)`?

e.g.
```
final LayoutInflater inflater = LayoutInflater.from(context);
final int blabla = mReaderImprovementsFeatureConfig.isEnabled() ? R.layout.reader_cardview_removed_post_new : R.layout.reader_cardview_removed_post;
postView = inflater.inflate(blabla, parent, false);
```